Library/Formula/rswift.rb

```diff
@@ -0,0 +1,17 @@
+class Rswift < Formula
+  desc "Get strong typed, autocompleted resources like images, fonts and segues"
+  homepage "https://github.com/mac-cain13/R.swift"
+  url "https://github.com/mac-cain13/R.swift.git",
+    :tag => "v1.1.1",
+    :revision => "3a6db62164d8f50ccdf43e59894fffb672fd5e3f"
+  depends_on :xcode => "7.0"
+
+  def install
+    xcodebuild "-configuration", "Release", "-scheme", "rswift", "SYMROOT=symroot", "OBJROOT=objroot"
+    bin.install "symroot/Release/rswift"
+  end
+
+  test do
+    system "rswift", "-h"
```

Comment:
Good point @bfontaine, @tomasharkema asked me to take a look at the possibilities for a better test.

I think it’s a bit hard to really use the tool in a test, since that would need a valid iOS Xcode project. Creating a fixture for that would be a bit of a hassle I think. 

The project itself tests if it output is correct using CI, so if we know that the binary is functional by executing it with a `-h` flag then I’m pretty sure everything is ok. I agree a test that actually uses the tool is better, but I think it’s not worth the hassle and maintenance in this case.

Also, I noted some other tools also use this kind of tests, so it seems like a good fallback to me.